seacrowd/sea_datasets/filipino_hatespeech_election/filipino_hatespeech_election.py

```diff
@@ -0,0 +1,124 @@
+import csv
+import os
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """
+@article{Cabasag-2019-hate-speech,
+  title={Hate speech in Philippine election-related tweets: Automatic detection and classification using natural language processing.},
+  author={Neil Vicente Cabasag, Vicente Raphael Chan, Sean Christian Lim, Mark Edward Gonzales, and Charibeth Cheng},
+  journal={Philippine Computing Journal},
+  volume={XIV},
+  number={1},
+  month={August},
+  year={2019}
+}
+"""
+
+_DATASETNAME = "filipino_hatespeech_election"
+
+_DESCRIPTION = """
+The dataset used in this study was a subset of the corpus 1,696,613 tweets crawled by Andrade et al. and posted from November 2015 to May 2016 during the campaign period for the Philippine presidential election. They were culled
+based on the presence of candidate names (e.g., Binay, Duterte, Poe, Roxas, and Santiago) and election-related hashtags (e.g., #Halalan2016, #Eleksyon2016, and #PiliPinas2016). Data preprocessing was performed to prepare the
+tweets for feature extraction and classification. It consisted of the following steps: data de-identification, uniform resource locator (URL) removal, special character processing, normalization, hashtag processing, and tokenization.
+"""
+
+_HOMEPAGE = "https://huggingface.co/datasets/hate_speech_filipino"
+
+_LANGUAGES = ["fil"]
+
+_LICENSE = Licenses.UNKNOWN.value
+
+_LOCAL = False
+
+_URLS = {_DATASETNAME: "https://s3.us-east-2.amazonaws.com/blaisecruz.com/datasets/hatenonhate/hatespeech_raw.zip"}
+
+_SUPPORTED_TASKS = [Tasks.SENTIMENT_ANALYSIS]
```

Comment:
consider remap it as `ABUSIVE_LANGUAGE_PREDICTION`

Reply:
Do you mean to change the task to `ABUSIVE_LANGUAGE_PREDICTION`?  but I am following https://github.com/SEACrowd/seacrowd-datahub/blob/master/seacrowd/sea_datasets/filipino_hatespeech_tiktok/filipino_hatespeech_tiktok.py#L31 and they should be the same task?

Reply:
@chenxwh @sabilmakbar I agree with you. Let's put it on our backlog. We should separate the `SENTIMENT_ANALYSIS` and `ABUSIVE_LANGUAGE_PREDICTION` dataloaders.

Reply:
sure! updated the task